soc/xtensa/intel_adsp/common/bootloader/boot_entry.S

```diff
@@ -16,10 +16,12 @@
 #include <devicetree.h>
 #include <soc/shim.h>
 #include <cavs-shim.h>
-#include <soc/platform.h>
 #include <soc/memory.h>
 #include <xtensa/corebits.h>
 #include <xtensa/config/core-isa.h>
+#include "platform.h"
+
+#define PLATFORM_PRIMARY_CORE_ID 0
```

Comment:
I don't see a mailbox.h in the tree?  Maybe that got removed somewhere else, or do you mean a SOF header?  Actually one of the side benefits to removing the old API headers is that we avoid tripping over spots where SOF was pulling in Zephyr symbols instead of using its own or moving to the Zephyr API (in this case the SMP CPU struct).